app/classifier/index.cjsx

```diff
@@ -250,17 +250,11 @@ Classifier = React.createClass
     </div>
 
   renderExpertOptions: ->
-    if @props.project?
-      getUserRoles = @props.project.get 'project_roles'
+    if @props.project? && @props.user?
+      getUserRoles = @props.project.get('project_roles', user_id: @props.user.id)
```

Comment:
that explains why i was seeing strange API request caching behaviour. Why won't it cache the API call results if it's the same API url?